README.md

```diff
@@ -98,7 +98,7 @@ Templates for homework: https://github.com/ChShersh/fp-homework-templates
   + **let** (variable declaration)
   + **where** clause
   + **if** expression
-  + **guards**
+  + Guards
   + **case** expression
   + Higher order functions
   + Lambdas (anonymous functions)
```

Comment:
It's better to add to course of this plan which extensions were used here: `-XTupleSections`, `-XLambdaCase` and `-XViewPatterns`. Also: `($)` and `(.)` should be formatted properly.